Functions/xASL_stat_GetROIstatistics.m

```diff
@@ -243,13 +227,14 @@
 bDoOnceROIStart = 1;
 
 for iSubject=1:x.nSubjects
+    
 	for iSess=1:nSessions
         
 		% ID (which name, group etc), all for identification
 		% Subject_session definition
 		DataIm = NaN;
 		SubjSess = (iSubject-1)* nSessions +iSess;
-		if NoSessions
+		if bSessionsMissing == 1

```

Comment:
The ` == 1` is superfluous

Reply:
I'll change it and merge! I've tested it already and it works.